src/store/chat/utils.ts

```diff
@@ -37,3 +37,24 @@ export function translateJoinRoomApiError(errorCode: JoinRoomApiErrorCode | stri
 
   return content;
 }
+
+// conversation can be referenced by an id or an alias
+export function isAlias(id: string) {
+  const isAlias = !id.startsWith('!');
+  if (isAlias) {
+    id = '#' + id;
+    return true;
+  }
+
+  return false;
```

Comment:
```suggestion
  return !id.startsWith('!');
```

Pretty sure the statement setting the variable isn't doing anything (and shouldn't because this is just a boolean check function)

Reply:
🤦‍♂️ i forgot to update this. Had added `parseAlias`. Thanks.